app/controllers/policies_controller.rb

```diff
@@ -67,6 +79,22 @@ def find_or_create_root_policy
 
   private
 
+  def audit_success(policy_version)
```

Comment:
the param here is `policy_version` bu all the calling methods use `policy`. can we align this?